### PR TITLE
Use encrypt API for enclave env secrets

### PR DESCRIPTION
**Pull request by doylemark**
# Why
After removing our internal rust crypto dependency, we removed this command from the codebase.

# How
Use the encrypt API to encrypt secret env vars.

### CODE REVIEW

File: crates/ev-cli/src/commands/enclave/mod.rs [hunk 1]
```diff
@@ -51,7 +53,7 @@ pub async fn run(enclave_args: EnclaveArgs) {
         #[cfg(not(target_os = "windows"))]
         EnclaveCommand::Attest(attest_args) => attest::run(attest_args).await,
         EnclaveCommand::Authenticated(authenticated_command) => {
-            let (api_key, _) = crate::get_auth();
+            let (app_uuid, api_key) = crate::get_auth();
```
Comment (doylemark) [inline crates/ev-cli/src/commands/enclave/mod.rs:56]: woops

File: crates/ev-enclave/src/env/mod.rs
Comment (doylemark) [inline crates/ev-enclave/src/env/mod.rs:1]: this is lifted off the original branch before it was deleted

File: crates/ev-cli/src/commands/enclave/env.rs [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    /// If the env var is a secret, it will be encrypted
+    #[clap(long = "secret")]
+    pub is_secret: bool,
+
+    /// Path to enclave.toml config file
+    #[clap(short = 'c', long = "config", default_value = "./enclave.toml")]
+    pub config: String,
+}
+
+/// Add delete secret from Enclave env
```
Comment (hanneary) [inline crates/ev-cli/src/commands/enclave/env.rs]:
```suggestion
/// Delete secret from Enclave env
```

File: crates/ev-enclave/src/env/mod.rs [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        || enclave_config.uuid.is_none()
+    {
+        Err(EnvError::MissingAppInfo)
+    } else {
+        Ok(EnclaveInfo {
+            app_uuid: enclave_config.app_uuid.unwrap(),
+            team_uuid: enclave_config.team_uuid.unwrap(),
+            uuid: enclave_config.uuid.unwrap(),
+        })
+    }
```
Comment (hanneary) [inline crates/ev-enclave/src/env/mod.rs]:
```suggestion
    let app_uuid = enclave_config.app_uuid.ok_or(EnvError::MissingAppInfo)?;
    let team_uuid = enclave_config.team_uuid.ok_or(EnvError::MissingAppInfo)?;
    let uuid = enclave_config.uuid.ok_or(EnvError::MissingAppInfo)?;
    
        Ok(EnclaveInfo {
            app_uuid,
            team_uuid,
            uuid
        })
    }
```

Comment (hanneary) [reply crates/ev-enclave/src/env/mod.rs]: the unwrap was never gonna be hit there but nice to get rid of it anyway

**Review by hanneary — COMMENTED** [review]
lgtm just minor nits, I noticed there's a few warnings on build would be good to clean them up too

**Review by hanneary — APPROVED** [review]

**Pull request merged**